routers/api/v1/repo/issue_tracked_time.go

```diff
@@ -212,6 +214,20 @@ func ListTrackedTimesByRepository(ctx *context.APIContext) {
 	//   description: name of the repo
 	//   type: string
 	//   required: true
+	// - name: user
+	//   in: query
+	//   description: optional filter by user
+	//   type: string
+	// - name: since
+	//   in: query
+	//   description: Only show times updated after the given time. This is a timestamp in ISO 8601 format
```

Comment:
Probably needs to be updated to match the Swagger definition of date-time: RFC3339